### DISCUSSION TITLE
MIP-6: Uses Feature Deprecation

### DISCUSSION THREAD

**Discussion by blockiosaurus** [metaplex-foundation/mip#27] [Announcements]
Discussion for [MIP-6: Uses Feature Deprecation](https://github.com/metaplex-foundation/mip/blob/main/mip-6.md)

**Comment by IvaldiS6** [comment]
I've always liked the option "Uses" made available, like a subscription token that will expire at some point.  I haven't gotten around to creating or using it.  

Is this something that could be added to pNFTs in their associated program(s)?
